### PR TITLE
Cleanup docs

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation
Review and update development documentation

## Progress

- [x] Validate new workflow after ansible broker update.
- [x] Move proposal (after @maleck13 approval)

### CODE REVIEW

File: docs/proposals/external-ns-service-integration.md [hunk 1]
```diff
@@ -1,53 +0,0 @@
-#External Service Integrations
```
Comment (wtrocki) [inline docs/proposals/external-ns-service-integration.md]: This can be moved to proposals repo. I can revert change if that's required in this repo.

**Comment by wtrocki** [comment]
@darahayes I have added some hints and tests. Ready for verification.

File: docs/walkthroughs/developing-apbs-locally.adoc [hunk 2]
```diff
@@ -17,16 +17,17 @@ alias apb='docker run --rm --privileged -v $PWD:/mnt -v $HOME/.kube:/.kube -v /v
 
 There is a link:https://github.com/ansibleplaybookbundle/ansible-playbook-bundle/blob/master/docs/getting_started.md[great tutorial] on how to write ABPs, it's highly recommended going through that!
 
-_NOTE:_ It's a good practise to have the APB end with `-apb`!
-
+To create apb execute
```
Comment (darahayes) [inline docs/walkthroughs/developing-apbs-locally.adoc]: `To create an APB execute`

**Comment by wtrocki** [comment]
@maleck13 @david-martin Can you approve or reject changes?

**Pull request merged**